_posts/en/newsletters/2021-02-10-newsletter.md

```diff
@@ -0,0 +1,231 @@
+---
+title: 'Bitcoin Optech Newsletter #135'
+permalink: /en/newsletters/2021/02/10/
+name: 2021-02-10-newsletter
+slug: 2021-02-10-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to the summary of last week's taproot
```

Comment:
```suggestion
This week's newsletter links to a summary of last week's taproot
```

Reply:
Other people have written summaries (e.g. Folkson's email links to Rusty Russell's toot summary), so I think "a" is the most appropriate article here.